website/src/components/Index/ContactUsSection/ContactUsSection.view.tsx

```diff
@@ -85,16 +85,21 @@ export default function ContactUsSection(): React.JSX.Element {
                         <button type="button" className="button button--outline button--primary button--lg"
                                 title={email as string}
                                 disabled={!isValid} onClick={handleSubmit(onSendEmail)}>
-                            📨 By Email
-                        </button>
-                        <button type="submit" className="button button--primary button--lg"
-                                title={newIssueUrl as string}
-                                disabled={!isValid}>
-                            💬 Using GitHub
+                            📨 Submit
                         </button>
                     </div>
                 </div>
             </form>
+	    <p>Or, join the <b>#fasttrackml</b> channel on the <b>MLOps.community</b> Slack!</p>
+        <div className="row margin-vert--md">
```

Comment:
Could all this javascript be replaced with a simple <a> tag and css to make it look like a button?

Reply:
i suppose it could but I was just following the pattern that exists here -- also, JS let's us reuse the url string in a few places.

Reply:
done, following Mehdi's suggestions